src/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Squoosh</title>
+		<meta name="viewport" content="width=device-width,initial-scale=1">
+		<meta name="mobile-web-app-capable" content="yes">
+		<meta name="apple-mobile-web-app-capable" content="yes">
+		<meta name="theme-color" content="#673ab8">
+		<link rel="manifest" href="/manifest.json">
+	</head>
+	<body>
+		<div id="app" prerender></div>
```

Comment:
Why invent an attribute here rather than use a class name (or a data attribute)?

Reply:
fixed in subsequent branch (prerender)